### PR TITLE
fix(rich-text): force trailing paragraph

### PR DESCRIPTION
**Pull request by z0al**
Used [@udecode/plate-trailing-block](https://github.com/udecode/plate/tree/main/packages/trailing-block) to keep an empty paragraph at the end of the document all the time. I think this behaviour matches the old editor IIRC.

Let me know if we have other custom logic somewhere that can be removed.

See [Forced the layout](https://plate.udecode.io/docs/plugins/forced-layout) docs. 

Let's wait for CI 😄

### CODE REVIEW

**Review by cribbles — APPROVED** [review]

**Review by cezarsmpio — APPROVED** [review]

**Comment by z0al** [comment]
Sadly, it completely breaks the list plugin at Cypress and I see some weird behavior Tables too 😞

File: packages/rich-text/src/RichTextEditor.tsx [hunk 1]
```diff
@@ -91,6 +92,9 @@ const getPlugins = (sdk: FieldExtensionSDK, tracking: TrackingProvider) => {
     createCodePlugin(),
     createItalicPlugin(),
     createUnderlinePlugin(),
+
+    // Other
+    createTrailingParagraphPlugin(),
```
Comment (cezarsmpio) [inline packages/rich-text/src/RichTextEditor.tsx:97]: The order matters here, shouldn't we add it to the top instead? So its rules will run before our plugins, but at the same time I'm afraid our plugins can override some of its rules 🤷‍♀️

Comment (z0al) [reply packages/rich-text/src/RichTextEditor.tsx:97]: 🤷🏻‍♂️

**Comment by z0al** [comment]
Fixed all issues except the hyperlink tests. some of the tests fail on Cypress even though seem to work just fine in the actual Cypress UI.

**Comment by z0al** [comment]
Wait it passed on Circle!!

what

**Review by cezarsmpio — APPROVED** [review]

**Pull request merged**